src/Arriba/Arriba.Server/Extensions/OAuthExtension.cs

```diff
@@ -2,22 +2,26 @@
 using Arriba.Configuration;
 using Microsoft.AspNetCore.Authentication.JwtBearer;
 using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Authorization.Infrastructure;
 using Microsoft.Extensions.DependencyInjection;
+using System.Linq;
+using System.Threading.Tasks;
 
 namespace Arriba.Extensions
 {
     public static class OAuthExtension
-    { 
+    {
         public static IServiceCollection AddOAuth(this IServiceCollection services, IArribaServerConfiguration serverConfig)
         {
+            services.AddSingleton<IAuthorizationHandler, AllowAnonymous>();
```

Comment:
This line confused me. I think it has todo do with the filter being called AllowAnonymous. I would suggest renaming maybe to **ArribaConfigurationAuthorizationOverride**. This indicates is something in this project.

Reply:
I did use ArribaConfigurationAuthorizationOverrideHandler at the handlers folder, also register it at Startup ConfigureService to be easy to find it